Packs/ArcherRSA/ReleaseNotes/1_1_26.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### RSA Archer v2
+- Edited the value of the parameter *Advanced: API Endpoint*.
```

Comment:
declare it as breaking change and explain what needs to be configured in each case:
* https://example.com/rsaarcher/api:
  * URL: https://example.com/rsaarcher
  * API Endpoint: /api
* https://example.com/api
  * URL: https://example.com/
  * API Endpoint: /api